test/error-placement.js

```diff
@@ -162,10 +162,10 @@ test( "errorcontainer, show/hide only on submit", function() {
 	ok( labelContainer.is( ":visible" ), "must be visible" );
 	ok( container.is( ":visible" ), "must be visible" );
 
-	$( "#firstname" ).val( "hix" ).keyup();
```

Comment:
Wdyt think about leaving keyup tests as is and add additional oninput tests.. That we we can more easily delete keyup coverage and we lso make sure that onkeyup handling works in browsers which support both